iosApp/iosApp/Pages/Map/HomeMapViewHandlerExtension.swift

```diff
@@ -82,8 +82,8 @@ extension HomeMapView {
     func joinVehiclesChannel(routeId: String, directionId: Int32) {
         leaveVehiclesChannel()
         vehiclesRepository.connect(routeId: routeId, directionId: directionId) { outcome in
-            if let data = outcome.data {
-                vehiclesData = Array(data.vehicles.values)
+            if case let .ok(data) = onEnum(of: outcome) {
+                vehiclesData = Array(data.data.vehicles.values)
```

Comment:
Sorry for the nitpick, but all the `data.data`s here and throughout make me sad and are a bit unreadable. Could we do something like `result.data` instead?

Reply:
Turned every `data.data` into a `result.data`.